Dockerfile

```diff
@@ -22,6 +22,12 @@
 
 FROM maven:3.6.0-jdk-8 as builder
 
+# Warmup to avoid downloading the world each time
+RUN git clone https://github.com/jenkinsci/plugin-compat-tester &&\
+    cd plugin-compat-tester && \
+    mvn clean package -DskipTests dependency:go-offline && \
```

Comment:
If you just want to warm the dependencies cache, only running `mvn dependency:go-offline` is enough.

Reply:
It can be tricky, given some modules may then fail for missing a dependent module. I suspect this may be why Oleg did this.

Reply:
Just copy-pasted from my another Dockerfile actually. Can retry the suggested option if you prefer @alecharp

Reply:
It's just that you package the project just to discard it right after. It's a waste of time IMO. Can you just try locally and if it doesn't work, I'll approve this.